scripts/add.models.sh

```diff
@@ -25,6 +25,8 @@ addModelFile "${FQDN}" "Preles"   "PRELES"   ""          "true"           "/bin"
 
 # special case for R models
 addModelFile "${FQDN}" "BioCro"   "BIOCRO"   "git"       "biocro.Rscript" "${R_LIBS_USER}/PEcAn.BIOCRO"
+addModelFile "${FQDN}" "BioCro"   "BIOCRO"   "0.95"      "biocro.Rscript" "${R_LIBS_USER}/PEcAn.BIOCRO"
+addModelFile "${FQDN}" "BioCro"   "BIOCRO"   "1.0"       "biocro.Rscript" "${R_LIBS_USER}/PEcAn.BIOCRO"
```

Comment:
@robkooper is this change reasonable? Same script gets called for both versions, we just use the Bety version record to keep track of which formats to give it.